pkg/ddevapp/router.go

```diff
@@ -102,7 +102,7 @@ func StartDdevRouter() error {
 	}
 
 	// run docker-compose up -d against the ddev-router full compose file
-	_, _, err = dockerutil.ComposeCmd([]string{routerComposeFullPath}, "-p", RouterProjectName, "up", "-d")
+	_, _, err = dockerutil.ComposeCmd([]string{routerComposeFullPath}, "-p", RouterProjectName, "up", "--build", "-d")
```

Comment:
Wondering if we also should use `--force-recreate` like for the SSH agent container or would this break something?